packages/dcos-launch/extra/launch/config.py

```diff
@@ -49,375 +21,256 @@ def expand_path(path: str, relative_dir: str) -> str:
     return os.path.abspath(os.path.join(relative_dir, path))
```

Comment:
Not your change. But  expand_path's scope is within LaunchValidator. This can be moved inside that class or it could be made a _private function.